jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -532,29 +517,17 @@ public void doStore(String id, SessionData data, long lastSaveTime) throws Excep
 
         sets.put(__ACCESSED, data.getAccessed());
         sets.put(__LAST_ACCESSED, data.getLastAccessed());
+        
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ObjectOutputStream oos = new ObjectOutputStream(baos);
+        SessionData.serializeAttributes(data, oos);
```

Comment:
`ObjectOutputStream` not closed, use try-with-resources.

Reply:
ok